views/optin.inc.php

```diff
@@ -0,0 +1,92 @@
+<?php
+namespace Langchecker;
+
+use \Bugzilla\Bugzilla;
+
+$htmloutput = '<p id="back"><a href="http://l10n.mozilla-community.org/webdashboard/">Back to Web Dashboard</a></p>';
```

Comment:
$html_output (I know we have other views where we use $htmloutput but let's stick to conventions for new content :) )